packages/token-manager/src/token-manager.ts

```diff
@@ -0,0 +1,68 @@
+import { TokenRecord } from '@accounts/common';
+
+import { randomBytes } from 'crypto';
+import * as jwt from 'jsonwebtoken';
+import { error } from 'util';
+
+import { Configuration } from './types/configuration';
+import { TokenGenerationConfiguration } from './types/token-generation-configuration';
+
+const defaultTokenConfig: TokenGenerationConfiguration = {
+  algorithm: 'HS256',
+};
+
+const defaultAccessTokenConfig: TokenGenerationConfiguration = {
+  expiresIn: '90m',
+};
+
+const defaultRefreshTokenConfig: TokenGenerationConfiguration = {
+  expiresIn: '7d',
+};
+
+export default class TokenManager {
+
+  private secret: string;
+
+  private emailTokenExpiration: number;
+
+  private accessTokenConfig: TokenGenerationConfiguration;
+
+  private refreshTokenConfig: TokenGenerationConfiguration;
+
+  constructor(config: Configuration) {
```

Comment:
what are your thoughts on just defining and exporting `Configuration` interface within this file and eliminating `./types` entirely?

Reply:
I want to enforce a pattern which has to be respected by all the packages of the suite, so if a potential contributor need to add a package, he will apply the pattern to his package naturally